matplotcheck/autograde.py

```diff
@@ -0,0 +1,89 @@
+# Wrapper functions for matplotcheck functions that run
+# and track the passing status of test functions and
+# provide formatted results of tests
+
+
+def run_test(
+    func,
+    points,
+    *args,
+    correct_message="default correct",
+    error_message="default error",
+    **kwargs
+):
+    """Runs a pre-defined test function and creates a dictionary
+    containing the results of the test
+
+    Parameters
+    ---------
```

Comment:
@jlpalomino did this actually build docs successfully? i'm asking because it looks like it is missing one `-` which is required for .Rst to render properly.

Reply:
The docs did build locally successfully.  Added the missing `-`.